index.js

```diff
@@ -34,8 +36,22 @@ module.exports = {
     return this._getBabelOptions(config);
   },
 
-  transpileTree(tree, config) {
-    return require('broccoli-babel-transpiler')(tree, this.buildBabelOptions(config));
+  _debugTree() {
+    if (!this._cachedDebugTree) {
+      this._cachedDebugTree = require('broccoli-debug').buildDebugCallback(`ember-cli-babel:${this._parentName()}`);
+    }
+
+    return this._cachedDebugTree.apply(null, arguments);
+  },
+
+  transpileTree(inputTree, config) {
+    let description = (`000` + (++count)).slice(-3);
```

Comment:
haha, indeed. thought about doing something smarter, but this seemed simple and I can't imagine the level of trollage if I actually add a new `leftpad` dependency 😝